unlock-app/src/components/interface/checkout/CryptoCheckout.tsx

```diff
@@ -255,15 +269,24 @@ export const CryptoCheckout = ({
             onRecipientChange={onRecipientChange}
             customBuyMessage="Buy for a different address"
             disabled={transactionPending?.length > 0 ?? false}
+            loading={loading}
           />
         </>
       )}
       {showCheckoutButtons && (
-        <div style={{ marginBottom: '32px' }}>
+        <div
```

Comment:
@searchableguy 

I'm passing the `loading` state from the child component to the parent, this causes a re-render. To present this issue i use inline style to temporally hide the component when is not needed